source/lmp/tests/model_convert.py

```
@@ -0,0 +1,126 @@
+            handle.write(f"pid={os.getpid()}\n")
+
+        if _is_up_to_date(source, output):
+            return output
+
+        tmp_fd, tmp_name = tempfile.mkstemp(
+            dir=output.parent,
+            prefix=f".{output.name}.",
+        )
+        os.close(tmp_fd)
+        tmp_path = Path(tmp_name)
+        sp.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)